package.json

```diff
@@ -29,13 +25,14 @@
     "react-native-paper": "^2.16.0",
     "react-native-vector-icons": "^6.6.0",
     "react-navigation": "^3.11.1",
+    "react-navigation-hooks": "^1.0.1",
     "react-redux": "^7.1.0",
     "redux": "^4.0.4",
     "redux-actions": "^2.6.5",
-    "redux-immutable-state-invariant": "^2.1.0",
     "redux-logger": "^3.0.6",
     "redux-persist": "^5.10.0",
     "redux-saga": "^1.0.5",
+    "redux-starter-kit": "https://github.com/ramyareye/redux-starter-kit",
```

Comment:
WHAT???!!! 😳

Reply:
I sent a PR and it's not merged yet, whenever they merged it I'll replace this package